packages/render-fidelity-tools/test/renderers/model-viewer/index.html

```diff
@@ -66,13 +66,13 @@
 
       modelViewer.src = scenario.model;
 
-      modelViewer.skyboxImage = null;
-      modelViewer.environmentImage = null;
       if (scenario.renderSkybox) {
+        // this set both skyboxImage and environmentImage to scenario.lighting;
```

Comment:
That's not quite true; they will only both be the same if environmentImage is set to null, so you should do that here as well, since you might be switching from a scenario with an environmentImage.